src/test/java/org/opensearch/sdk/TestExtensionsRunner.java

```diff
@@ -113,8 +113,8 @@ public void testHandleExtensionInitRequest() throws UnknownHostException {
             "opensearch-sdk-1",
             sourceNode.getAddress(),
             new HashMap<String, String>(),
-            Version.fromString("3.0.0"),
-            Version.fromString("3.0.0"),
+            Version.fromString("2.6.0"),
+            Version.fromString("2.6.0"),
```

Comment:
Eww, not only do we need to change the dependency with import, we need to change code as well?  Or does any 2.x version work in this test?   

Might consider creating a system property in the `build.gradle` and pulling it in from here.

Reply:
That's a good idea

Reply:
Potential solution in the comment above: https://github.com/opensearch-project/opensearch-sdk-java/pull/420#discussion_r1097997733

Reply:
I'm going to guess the exact version doesn't matter and we can leave this as is.

Reply:
Actually there's an equals comparison with CURRENT, I think.

So this would have been a nice red flag that we needed to update our version, but why not just set these lines to `Version.CURRENT`?

Reply:
It's minimum compatibility so it'll pass as long as the OpenSearch version > than 2.6

Reply:
I don't see any reason not to change it to `Version.CURRENT` though, so I did